test/cowboy_swagger_SUITE.erl

```diff
@@ -173,6 +177,42 @@ schema_test(_Config) ->
 %% Internal functions
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+perform_add_completed_definition_test() ->
+  %%
+  %% Given
+  %%
+  ct:comment("Add first definition"),
+  Name1 = <<"CostumerDefinition">>,
+  Properties1 = test_properties_one(),
+  Definition1 =
+    #{ Name1 =>
+      #{ type => <<"object">>
+       , properties => Properties1}
+    },
+
+  ct:comment("Add second definition"),
+  Name2 = <<"CarDefinition">>,
+  Properties2 = test_properties_two(),
+  Definition2 =
+    #{ Name1 =>
+      #{ type => <<"object">>
+       , properties => Properties2}
+    },
+  %%
+  %% When
+  %%
+  ok = cowboy_swagger:add_definition(Definition1),
+  ok = cowboy_swagger:add_definition(Definition2),
+
+  %%
+  %% Then
+  %%
+  {ok, SwaggerSpec1} = application:get_env(cowboy_swagger, global_spec),
+  JsonDefinitions = cowboy_swagger:get_existing_definitions(SwaggerSpec1),
+  true = maps:is_key(Name1, JsonDefinitions),
+  true = maps:is_key(Name2, JsonDefinitions),
+  ok.
```

Comment:
```suggestion
  {comment, ""}.
```
If your test case uses `ct:comment/1,2`, it must return `{comment, ""}` when it's successful.